console/module/tag/src/main/java/org/eclipse/kapua/app/console/module/tag/client/TagAddDialog.java

```diff
@@ -49,6 +49,7 @@ public void createBody() {
         tagNameField.setAllowBlank(false);
         tagNameField.setFieldLabel("* " + MSGS.dialogAddFieldName());
         tagNameField.setValidator(new TextFieldValidator(tagNameField, FieldType.NAME));
+        tagNameField.setMaxLength(255);
```

Comment:
This is fine, but please use `org.eclipse.kapua.app.console.module.api.client.ui.widget.KapuaTextField`.

It contains a small fix for the handling of the validation